### ISSUE TITLE
Fatal Python error: Cannot recover from stack overflow.

### ISSUE THREAD

**Issue by anirudhitagi** [facebookresearch/HRViT#8]
Greetings,

I am trying to train the HRViT_b1_224 model and I got the following error: 
[hrvit_error_log.txt](https://github.com/facebookresearch/HRViT/files/10745088/hrvit_error_log.txt)

Here is the command I used to run the training script: `$ bash train.sh 1 1 0 --data ../data/imagenet_1k/ --model HRViT_b1_224 -b 32 --lr 1e-3 --weight-decay .05 --amp --img-size 224 --warmup-epochs 20 --drop-path 0.1 --head-drop 0.1 --clip-grad 1 --sync-bn --with-cp`

1. Could there be an issue with running the training on 1 GPU only?  or 
2. could this be an issue because I am training it on a much smaller data set? [ I have maintained the same folder structure as instructed : 
Could either of these be the issue? 
```
└── imagenet_1k
    ├── labels.txt
    ├── synset_words.txt
    ├── train
    │   ├── n01440764
    │   │   └── ILSVRC2012_00009111.JPEG
    │   ├── n01443537
    │   │   └── ILSVRC2012_00004677.JPEG
    │   ├── n01484850
    │   │   └── ILSVRC2012_00027419.JPEG
    │   ├── n01491361
    │   │   └── ILSVRC2012_00034413.JPEG
    │   ├── ............................
    │   │   ............................
    └── val
        ├── n01440764
        │   └── ILSVRC2012_val_00000293.JPEG
        ├── n01443537
        │   └── ILSVRC2012_val_00048840.JPEG
        ├── n01484850
        │   └── ILSVRC2012_val_00046171.JPEG
        ├── n01491361
    │   ├── ............................
    │   │   ............................
```

Please advise how to resolve this issue?
